src/Kovan-DssSpell.sol

```diff
@@ -95,148 +62,143 @@ contract SpellAction {
+        MedianAbstract(LPOsmAbstract(PIP_UNIV2USDCETH).orb1()).kiss(PIP_UNIV2USDCETH);
+        // Whitelist Spotter to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_UNIV2USDCETH = Osm or PIP_UNIV2USDCETH = Median and hasn't been already whitelisted due a previous deployed ilk
+        LPOsmAbstract(PIP_UNIV2USDCETH).kiss(MCD_SPOT);
+        // Whitelist End to read the Osm data (only necessary if it is the first time the token is being added to an ilk)
+        // !!!!!!!! Only if PIP_UNIV2USDCETH = Osm or PIP_UNIV2USDCETH = Median and hasn't been already whitelisted due a previous deployed ilk
+        LPOsmAbstract(PIP_UNIV2USDCETH).kiss(MCD_END);
+        // Set UNIV2USDCETH Osm in the OsmMom for new ilk
+        // !!!!!!!! Only if PIP_UNIV2USDCETH = Osm
+        OsmMomAbstract(OSM_MOM).setOsm(ILK_UNIV2USDCETH_A, PIP_UNIV2USDCETH);
+
+        // Set the UNIV2USDCETH-A debt ceiling
+        VatAbstract(MCD_VAT).file(ILK_UNIV2USDCETH_A, "line", 10 * MILLION * RAD);
+        // Set the UNIV2USDCETH-A dust
+        VatAbstract(MCD_VAT).file(ILK_UNIV2USDCETH_A, "dust", 100 * RAD);
         // Set the Lot size
-        CatAbstract(MCD_CAT).file(ILK_PSM_USDC_A, "dunk", 500 * RAD);
-        // Set the PSM-USDC-A liquidation penalty (e.g. 13% => X = 113)
-        CatAbstract(MCD_CAT).file(ILK_PSM_USDC_A, "chop", 113 * WAD / 100);
-        // Set the PSM-USDC-A stability fee (e.g. 1% = 1000000000315522921573372069)
-        JugAbstract(MCD_JUG).file(ILK_PSM_USDC_A, "duty", ZERO_PERCENT_RATE);
-        // Set the PSM-USDC-A percentage between bids (e.g. 3% => X = 103)
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("beg", 103 * WAD / 100);
-        // Set the PSM-USDC-A time max time between bids
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("ttl", 1 hours);
-        // Set the PSM-USDC-A max auction duration to
-        FlipAbstract(MCD_FLIP_USDC_PSM).file("tau", 1 hours);
-        // Set the PSM-USDC-A min collateralization ratio (e.g. 150% => X = 150)
-        SpotAbstract(MCD_SPOT).file(ILK_PSM_USDC_A, "mat", 100 * RAY / 100);
-        // Set the PSM-USDC-A fee in (tin)
-        PsmAbstract(MCD_PSM_USDC_PSM).file("tin", 1 * WAD / 100);
-        // Set the PSM-USDC-A fee out (tout)
-        PsmAbstract(MCD_PSM_USDC_PSM).file("tout", 1 * WAD / 1000);
-
-        // Update PSM-USDC-A spot value in Vat
-        SpotAbstract(MCD_SPOT).poke(ILK_PSM_USDC_A);
+        CatAbstract(MCD_CAT).file(ILK_UNIV2USDCETH_A, "dunk", 500 * RAD);
+        // Set the UNIV2USDCETH-A liquidation penalty (e.g. 13% => X = 113)
+        CatAbstract(MCD_CAT).file(ILK_UNIV2USDCETH_A, "chop", 113 * WAD / 100);
+        // Set the UNIV2USDCETH-A stability fee (e.g. 1% = 1000000000315522921573372069)
+        JugAbstract(MCD_JUG).file(ILK_UNIV2USDCETH_A, "duty", ONE_PERCENT_RATE);
```

Comment:
@godsflaw Do we already have an official risk list for these values? So excepting `dust`, `dunk`, `ttl` and `tau` we should try to set the same ones that will go to mainnet. Otherwise let's leave them as they are.

Reply:
I think we do: https://forum.makerdao.com/t/uni-v2-usdc-eth-collateral-onboarding-risk-evaluation/5335